src/log.py

```diff
@@ -87,20 +87,21 @@ def _update_log_parent_symlinks():
 
     one_year_ago = time.time() - 365 * 24 * 3600
 
-    logs = glob.iglob(OPERATIONS_PATH + "*" + METADATA_FILE_EXT)
+    logs = glob.iglob("*" + METADATA_FILE_EXT, root_dir=OPERATIONS_PATH)
     for log_md in logs:
-        if os.path.getctime(log_md) < one_year_ago:
+        log_file = os.path.join(OPERATIONS_PATH, log_md)
+        if os.path.getctime(log_file) < one_year_ago:
             # Let's ignore files older than one year because hmpf reading a shitload of yml is not free
             continue
 
         name = log_md[: -len(METADATA_FILE_EXT)]
-        parent_symlink = os.path.join(OPERATIONS_PATH, f".{name}.parent.yml")
-        if os.path.islink(parent_symlink):
+        parent_symlink = os.path.join(OPERATIONS_PATH, f"{name}.parent.yml")
```

Comment:
the issue with parents being displayed might be related to the fact that the file was meant to be hidden (prefixed with a dot) and maybe "glob" doesn't list hidden files by default, if it behaves the same as `ls`, not sure

Reply:
Hoo that dot was intended

Reply:
`include_hidden=True` param has been added to the glob function in python 3.11